test/client/spec/components/victory-line/victory-line.spec.js

```diff
@@ -77,6 +80,39 @@ describe("components/victory-line", () => {
       expect(lines.length).to.equal(1);
     });
 
+    it("renders a path with the correct d3Shape d attribute", () => {
+      const props = {
+        interpolation: "linear",
+        scale: "linear",
+        padding: 50,
+        width: 400,
+        height: 300,
+        data: [{x: 0, y: 0}, {x: 1, y: 1}, {x: 2, y: 2}]
+      };
+      const wrapper = shallow(
+        <VictoryLine {...props}/>
+      );
+
+      const line = wrapper.find(Line);
+      const path = $(line.html()).attr("d");
+
+      const scaleX = d3Scale.scaleLinear()
```

Comment:
Can you extract this into an helper since it will be used by VictoryArea specs as well